### PR TITLE
adding some touches

### PR DESCRIPTION
**Pull request by omarsaffo**
changes :
1- General UI and front-end changes.
2- Used Bootstrap library.
2- Added a popup message saying "Item added " when adding items.
3- Added an "Empty box" feature which detects if the user pressed submit with an empty custom order box and doesn't add the item and displays a popup message saying "box is empty".

### CODE REVIEW

File: writer/app.js [hunk 1]
```diff
@@ -1,95 +1,98 @@
...
-
-
-
-    
-  // Function to send the shopping list to the server
-    function sendShoppingList() {
-        const outputArea = document.getElementById('outputArea');
-        const shoppingListItems = Array.from(outputArea.children)
-            .map(item => item.firstChild.textContent.trim());
-
```
Comment (saffo7boss) [inline writer/app.js:84]: why are you removing the send-list API

File: writer/writer.html [hunk 2]
```diff
@@ -7,102 +7,161 @@
...
+
+        @keyframes fadeInOut {
+            0%, 100% {
+                opacity: 0;
+            }
+            10%, 90% {
+                opacity: 1;
+            }
+        }
+    </style>
```
Comment (saffo7boss) [inline writer/writer.html:35]: this css is also included in the stylesheet

File: writer/app.js (hunk 1 above)
Comment (omarsaffo) [inline writer/app.js:84]: didnt meant to ! probably deleted it by mistake

File: home/home.html [hunk 3]
```diff
@@ -6,47 +6,74 @@
...
+          <li class="nav-item">
+            <a class="nav-link" href="../contact-us/contact-us.html">contact</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="..\shopper\shopper.html">shopper</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="..\writer\writer.html">writer</a>
+          </li>
+        </ul>
```
Comment (saffo7boss) [inline home/home.html:37]: the hrefs don't work while running on the server. They only work while the server is offline mode and the file is run as an html file in the browser

File: shopper/style.css [hunk 4]
```diff
@@ -18,10 +19,11 @@ nav ul {
     margin: 0;
 }
 
-nav ul li {
+nav li {
     display: inline;
-    margin-right: 10px;
+    margin-right: 20px;
 }
+
```
Comment (saffo7boss) [inline shopper/style.css]: nav li and nav a seem to both affect the same area (the list of the hyperlinks in the footer)

File: writer/app.js (hunk 1 above)
Comment (saffo7boss) [inline writer/app.js:84]: why remove the send-list API?

File: writer/writer.html (hunk 2 above)
Comment (saffo7boss) [inline writer/writer.html]: this css is already included in the referred css file (line 8)

**Review by saffo7boss — COMMENTED** [review]
I like some of the new button designs, but the fonts and the white backgrounds instead of the grey not so much. There are also some elements, like the heading in the shopper html, that gotten closer to the header than they were before.
styling aside, the error handling of the empty custom order submission is great, not to mention the pop up of "item added to list" is amazing. But major problem is that the website isn't run-able on the server, meaning if you initialize the server and try to access it on port 3000 it will only run the home.html, but the rest won't work, I encountered this error before, the problem was with the specified paths, but since you didn't change them (much) I don't know what is wrong, but I know that they don't work. Since the writer isn't accessible, I wasn't able to check if sending the shopping list features is still working, but I doubt it since you deleted the whole function anyway.

File: shopper/style.css (hunk 4 above)
Comment (omarsaffo) [reply shopper/style.css]: I didnt miss with any of these😅 probably yours

**Review by omarsaffo — COMMENTED** [review]
added lines back in the writer
made UI configurations to be compatible with smaller devices
added a button to go to the  shopper after sending the List
fixed the list-sending error

**Comment by saffo7boss** [comment]
Awesome work, though unfortunately, contact us doesn't work on the phone. Everything else is fine

**Pull request merged**